checker/tests/nullness/Issue1142.java

```diff
@@ -0,0 +1,14 @@
+// Issue 1142 https://github.com/typetools/checker-framework/issues/67
+import java.util.concurrent.ConcurrentHashMap;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class Issue1142 {
+
+    void foo() {
+
+        ConcurrentHashMap<Integer, @Nullable Integer> chm = new ConcurrentHashMap<>();
```

Comment:
The test should pass exactly if the error is thrown.  Right now, you have written a test that passes only if the error is not thrown.

Please read the instructions for writing a test case, which appear in file `checker/tests/README`.